### PR TITLE
bt-182: Add logo component

### LINKED ISSUE #182: Add logo component

### PR DESCRIPTION
**Pull request by Monyster**
Closes #182 

Example code:
```
            <Logo />
            <Logo isCollapsed />
```

Result:
image

### CODE REVIEW

File: frontend/src/bundles/common/components/logo/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@import url("../../../../assets/css/styles.scss");
```
Comment (vladjsp) [inline frontend/src/bundles/common/components/logo/styles.module.scss]: we don't need this import here.

File: frontend/src/bundles/common/components/logo/logo.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    isCollapsed?: boolean;
+};
+
+const Logo = ({ isCollapsed = false }: Properties): JSX.Element => {
+    return (
+        <Grid container className={styles.logo}>
+            <Grid
+                item
+                alignItems="center"
+                className={clsx(
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/logo/logo.tsx]: use `getValidClassNames` helper

**Comment by vladjsp** [comment]
also seems that the big logo (67x67) has border-radius 7px and small one (48x48) has border-radius 5px

File: frontend/src/bundles/common/components/logo/logo.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import clsx from 'clsx';
+
+import logoSvg from '~/assets/img/logo/logo.svg';
+import logoLabelSvg from '~/assets/img/logo/logo-label.svg';
+
+import { Grid } from '../components.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+    isCollapsed?: boolean;
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/logo/logo.tsx:10]: add `className` prop to apply styles

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

**Comment by vladjsp** [comment]
in general looks good and easy to use
`<Logo isCollapsed={true}/>`
logo

**Review by nikita-remeslov — APPROVED** [review]

**Review by anton-otroshchenko — APPROVED** [review]

**Pull request merged**